src/azurelinux/3.0/net9.0/android/Dockerfile

```diff
@@ -5,9 +5,10 @@ RUN tdnf update -y \
     && tdnf install -y \
         # Common dependencies
         binutils \
+        pigz \
```

Comment:
I think I'd prefer installing it in the base image instead

Reply:
I installed it next to unzip, maybe that's not the right criteria?

Reply:
yeah we should really only need it in a very small subset of images

Reply:
Yup, updated the root ones, without touching helix stuff.